### PR TITLE
Added subcategories for supported titles

### PR DESCRIPTION
**Pull request by HyperBE32**
This PR updates the categories for all codes, where necessary, to move them into new subcategories to organise things better in their parent categories.

This PR depends on https://github.com/hedge-dev/HedgeModManager/pull/1 being merged, as well as a new HMM release being made to ensure this change is met across all versions of the mod manager.

Feedback on these subcategories is very much appreciated in the meantime.

### CODE REVIEW

**Comment by DeaTh-G** [comment]
Added a blocked label to further indicate the the PR can not be merged at the moment.

File: Source/Sonic Lost World/Fixes/Animation/Fix Super Sonic Mouth Switching.hmm [hunk 1]
```diff
@@ -1,578 +1,578 @@
...
-    originalReplace:
-        push ebp
-        mov ebp, esp
-        sub esp, 0x3C
-        mov eax, returnAddress
-        jmp eax
-    ",
-    ASLR(0x00BFDC10), HookBehavior.Replace
-)
+Patch "Fix Super Sonic Mouth Switching" in "Fixes/Animation" by "ĐeäTh & Sajid" does
```
Comment (DeaTh-G) [inline Source/Sonic Lost World/Fixes/Animation/Fix Super Sonic Mouth Switching.hmm:1]: I think this would be much better suited to be in Fixes/Graphics, much like `Fix Parkour Trail Model`.

File: Source/Sonic Lost World/Graphics/Display/Aspect Ratio/16-by-10.hmm [hunk 2]
```diff
@@ -0,0 +1,4 @@
+Patch "16:10" in "Graphics/Display/Aspect Ratio" by "ĐeäTh" does "Changes the game in-game resolution to a 16:10 aspect ratio. Perfect for the Steam Deck."
```
Comment (DeaTh-G) [inline Source/Sonic Lost World/Graphics/Display/Aspect Ratio/16-by-10.hmm:1]: I'm not sure if these fit right under the Aspect Ratio category, as they don't directly influence the game's output aspect ratio. These codes are similar to changing the internal resolution of Sonic Unleashed.

File: Source/Sonic Origins/Audio/Music/Sonic 3 & Knuckles/Disable Super Music.hmm [hunk 3]
```diff
@@ -0,0 +1,11 @@
+Code "Disable Super Music" in "Audio/Music/Sonic 3 & Knuckles" by "MegAmi" does "Due to how the cheat works, the Super music may still play in some instances, such as after the AIZ Act 1 bombing cutscene."
```
Comment (thesupersonic16) [inline Source/Sonic Origins/Audio/Music/Sonic 3 & Knuckles/Disable Super Music.hmm:1]: Do we really need /Music/ at this point of time? Having to open a subcategory for what might be a single code seems necessary to me. What do you think?

**Review by thesupersonic16 — COMMENTED** [review]
I am not so sure with a lot of the codes being in the Items category, going from the names alone some sounds more like Player related codes. For example "Disable Boost Drain" and "Wispon Gauge Stay's Full" being "Cheats/Items". I am not so sure people will see boost as an item, but more part of the player itself. Is there a reason why it's labeled this way?

File: Source/Sonic Origins/Audio/Music/Sonic 3 & Knuckles/Disable Super Music.hmm (hunk 3 above)
Comment (DeaTh-G) [reply Source/Sonic Origins/Audio/Music/Sonic 3 & Knuckles/Disable Super Music.hmm:1]: Do you mean unnecessary?

Comment (thesupersonic16) [reply Source/Sonic Origins/Audio/Music/Sonic 3 & Knuckles/Disable Super Music.hmm:1]: Yes. Not sure why I mess up my negatives so often.

**Comment by DeaTh-G** [comment]
As @thesupersonic16 has mentioned regarding an Origins code, I do think that a lot of single code sub-categories are kind of, unnecessary. This is extrapolated in games with more codes. Whilst it works for future proofing, we also don't really have to future proof as its not like the codes can't be moved into a sub category of a main category later down the line. Things like `Post-processing`, `Skills` and `Voice` here feel completely unnecessary, with each containing a single code and just more hoops for the end user to jump through to get to the code they would like to use.
image

**Comment by thesupersonic16** [comment]
> As @thesupersonic16 has mentioned regarding an Origins code, I do think that a lot of single code sub-categories are kind of, unnecessary. This is extrapolated in games with more codes. Whilst it works for future proofing, we also don't really have to future proof as its not like the codes can't be moved into a sub category of a main category later down the line. Things like `Post-processing`, `Skills` and `Voice` here feel completely unnecessary, with each containing a single code and just more hoops for the end user to jump through to get to the code they would like to use. image

Ah didn't realise there being that many `Audio` codes, but yeah subcategories with only one code seems unnecessary, maybe we could consider the UI to ignore them? Assuming it doesn't already.

**Comment by DeaTh-G** [comment]
> Ah didn't realise there being that many `Audio` codes, but yeah subcategories with only one code seems unnecessary, maybe we could consider the UI to ignore them? Assuming it doesn't already.

Well, that's not a really a solution either, as there are cases where that single code should fall into a sub-category, such as here:
image

However in my previous example its not particularly necessary to make individual sub-category for those 3 codes and could just fall into the main `Audio` category. Once more codes like them are made that would fit that sub-category they can be moved to a sub-category, as that point it would actually make sense.

File: Source/Sonic Lost World/Graphics/Display/Aspect Ratio/16-by-10.hmm (hunk 2 above)
Comment (HyperBE32) [reply Source/Sonic Lost World/Graphics/Display/Aspect Ratio/16-by-10.hmm:1]: Understood, should I move these into a `Resolution` subcategory and leave `Widescreen Support` as is?

**Comment by HyperBE32** [comment]
Agreed with all statements above, the `Audio/Post-processing`, `Audio/Skills` and `Audio/Voice` categories were a last minute addition to keep consistency between games. I'll revert these to prevent unnecessary nesting.

**Comment by thesupersonic16** [comment]
I am not a fan of the `Item` categories, at a glance, they do not make any sense to me as my mind thinks of objects in a stage, not that you froze the boost to 100%. I get boost and rings can be gained by items, but when used as unlimited/frozen I don't think they are items anymore especially since its held by the Player.

**Comment by DeaTh-G** [comment]
Some other places where nesting feels unnecessary to me:
`Sonic Colours Ultimate/Cheats`
`Sonic Colours Ultimate/Gameplay/Objects`
`Sonic Forces/Cheats/Skills`
`Sonic Frontiers/Cheats/Objects`
`Sonic Frontiers/Cheats/Player`
`Sonic Frontiers/Enemy/Boss`
`Sonic Frontiers/Fixes/Camera/Photo Mode`
`Sonic Frontiers/Fixes/Physics`
`Sonic Frontiers/Gameplay/Open Zone/Event`
`Sonic Generations/Gameplay/Enemy`
`Sonic Generations/Gameplay/Event`
`Sonic Lost World/Audio/Music`
`Sonic Origins/UI/Frontend`

**Comment by DeaTh-G** [comment]
> I am not a fan of the `Item` categories, at a glance, they do not make any sense to me as my mind thinks of objects in a stage, not that you froze the boost to 100%. I get boost and rings can be gained by items, but when used as unlimited/frozen I don't think they are items anymore especially since its held by the Player.

Yeah, I agree with this, codes such as `Sonic Forces/Cheats/Items/Disable Boost Drain.hmm`, `Sonic Forces/Cheats/Items/Disable Ring Loss from Damage.hmm`, `Sonic Forces/Cheats/Items/Disable Ring Loss from Super Sonic.hmm` don't sit right with me under an `Items` category.

**Comment by HyperBE32** [comment]
Yeah, I wasn't completely sure on where to put those to be honest, so I lumped them in with being "Items". Do we agree to move boost and Super form related codes to the "Player" subcategory?

**Comment by DeaTh-G** [comment]
Sounds good to me.

File: Source/Sonic Lost World/Graphics/Display/Aspect Ratio/16-by-10.hmm (hunk 2 above)
Comment (DeaTh-G) [reply Source/Sonic Lost World/Graphics/Display/Aspect Ratio/16-by-10.hmm:1]: I think putting `Widescreen Support` in `Sonic Lost World/Graphics/Display` is good enough, no need for a sub-category inside Display for it and put the other codes into `Sonic Lost World/Graphics/Display/Internal Resolution`. Having said that I'm not even sure if that amount of nesting is necessary and we could just go for `Sonic Lost World/Graphics/Widescreen Support.hmm` and `Sonic Lost World/Graphics/Internal Resolution` for the other codes instead.

Comment (HyperBE32) [reply Source/Sonic Lost World/Graphics/Display/Aspect Ratio/16-by-10.hmm:1]: I'll settle with `Graphics/Display/Widescreen Support` and `Graphics/Display/Internal Resolution`, however the names of these codes are a little confusing, as 16:10 and 32:9 are somewhat vague.

Comment (DeaTh-G) [reply Source/Sonic Lost World/Graphics/Display/Aspect Ratio/16-by-10.hmm:1]: Not sure what could be done about that in particular, as the game is using an aspect ratio value for the internal resolution and the codes just inject the aspect ratio values to that memory address.
https://www.wsgf.org/article/common-hex-values

Comment (HyperBE32) [reply Source/Sonic Lost World/Graphics/Display/Aspect Ratio/16-by-10.hmm:1]: I'll leave them as-is for now, might be worth revisiting these codes at a later point to make this a dynamic change, rather than having multiple codes.

Comment (DeaTh-G) [reply Source/Sonic Lost World/Graphics/Display/Aspect Ratio/16-by-10.hmm:1]: Ideally once configurable codes are a thing you could inject your custom resolution that gets calculated into an aspect ratio.

**Comment by DeaTh-G** [comment]
Regarding the `Items` and `Player` categories found in many of the games that we have `Cheats` for, I feel like there's a bit of a confusion on what those categories should mean.
Codes such as `Sonic Forces/Cheats/Items/Disable Wispon Boundaries.hmm` feels a bit misguided in `Items` category as that to me implies physical, in the world collectibles as opposed to interacting with inherent game mechanics, such as where the wispon is usable or not, thus it feels like it would fall into the `Player` sub-category.
Codes that do feel fit for the `Items` sub-category are codes such as `Sonic Frontiers/Cheats/Items/Keep Red Rings on Death` as it applies to how the item is found in the Cyber Space levels themselves.

Bottom line is, cheat codes affecting mechanics, such as the player's ring counter, boost, life count, etc. should be categorized under `Cheats/Player`, whilst codes affecting how the physical object found in the game as an interactable behaves would belong under `Cheats/Items`

**Comment by HyperBE32** [comment]
To prevent further unnecessary nesting, we could also just drop the `Items` subcategory and move everything to `Player` instead - would avoid any confusion later down the line, both to end-users and code developers on where things belong.

**Comment by DeaTh-G** [comment]
That solution is also fine with me (especially since once that ideology is applied I think there would be only 2-3 codes fit for `Items` category to begin with).

**Review by thesupersonic16 — APPROVED** [review]
From what I seen, looks good to me.

**Comment by ActualMandM** [comment]
[Hedge Mod Manager 7.12](https://github.com/thesupersonic16/HedgeModManager/releases/tag/7.12) is out now.

File: Source/Sonic Generations/Gameplay/Input/Disable Homing Delay.hmm [hunk 4]
```diff
@@ -1,4 +1,4 @@
-Patch "Disable Homing Delay" in "Gameplay" by "Hyper" 
+Patch "Disable Homing Delay" in "Gameplay/Input" by "Hyper"
```
Comment (ActualMandM) [inline Source/Sonic Generations/Gameplay/Input/Disable Homing Delay.hmm:1]: I think this would be better under `Gameplay/Player` given the rest of the Input subcategory codes are tied towards controller input. (button remapping, changing stick deadzone, disabling vibration)

**Comment by DeaTh-G** [comment]
Since I don't have time (due to personal matters) atm to personally go through the PR to re-review and approve/request changes I'll do some management related stuff regarding this PR.

The codes for the PR should be reviewed as the following:
@thesupersonic16 
- Sonic Forces
- Tokyo Olympics 2020
- Sonic Colors Ultimate
- Sonic Origins

@Sajidur78 
- Sonic Lost World

@blueskythlikesclouds 
- Sonic Generations

Please look for any unnecessary nesting or straight up wrong categories and approve or request changes to the codes for each game.
Frontiers can be seen as approved by @HyperBE32 seeing as he is the main maintainer of those codes but if you wish to you can go through those too.

File: Source/Sonic Frontiers/Graphics/Post-processing/Disable Color Correction LUTs.hmm [hunk 5]
```diff
@@ -1,4 +1,4 @@
-Patch "Disable Color Correction LUTs" in "Graphics" by "M&M" does "Prevents the game from using the color corrected LUTs stored in each stage's trr_cmn pac." 
+Patch "Disable Color Correction LUTs" in "Graphics/Post-processing" by "M&M" does "Prevents the game from using the color corrected LUTs stored in each stage's trr_cmn pac."
```
Comment (ActualMandM) [inline Source/Sonic Frontiers/Graphics/Post-processing/Disable Color Correction LUTs.hmm:1]: Color -> Colour

File: Source/Sonic Generations/Cheats/Hub/Always Unlocked All Stages.hmm [hunk 6]
```diff
@@ -1,4 +1,4 @@
-Patch "All Levels are Selectable" in "Cheats" by "Muzzarino" 
+Patch "Always Unlocked All Stages" in "Cheats/Hub" by "Muzzarino"
```
Comment (ActualMandM) [inline Source/Sonic Generations/Cheats/Hub/Always Unlocked All Stages.hmm:1]: Hub -> White Space

File: Source/Sonic Generations/Cheats/Hub/Disable Boss Key Check.hmm [hunk 7]
```diff
@@ -1,4 +1,4 @@
-Patch "Bypass Boss Key Check" in "Cheats" by "Muzzarino" does "Allows progression through White World without collecting the boss keys." 
+Patch "Disable Boss Key Check" in "Cheats/Hub" by "Muzzarino" does "Allows progression through White World without collecting the boss keys."
```
Comment (ActualMandM) [inline Source/Sonic Generations/Cheats/Hub/Disable Boss Key Check.hmm:1]: Hub -> White Space

File: Source/Sonic Generations/Gameplay/Objects/Disable Out Of Control Balloons.hmm [hunk 8]
```diff
@@ -0,0 +1,6 @@
+Patch "Disable Out Of Control Balloons" in "Gameplay/Objects" by "N69" does "Prevents control being restricted after hitting a balloon."
```
Comment (ActualMandM) [inline Source/Sonic Generations/Gameplay/Objects/Disable Out Of Control Balloons.hmm:1]: Out Of Control -> Out of Control

File: Source/Sonic Generations/Gameplay/Objects/Disable Out Of Control Jump Springs.hmm [hunk 9]
```diff
@@ -0,0 +1,6 @@
+Patch "Disable Out Of Control Jump Springs" in "Gameplay/Objects" does "Prevents control being restricted after hitting a spring."
```
Comment (ActualMandM) [inline Source/Sonic Generations/Gameplay/Objects/Disable Out Of Control Jump Springs.hmm:1]: Out Of Control -> Out of Control

File: Source/Sonic Generations/Gameplay/Speed/Everything is 2x Speed Except For Sonic.hmm [hunk 10]
```diff
@@ -1,4 +1,4 @@
-Code "Everything is 2x Speed Except For Sonic" in "Gameplay" by "Sajid" 
+Code "Everything is 2x Speed Except For Sonic" in "Gameplay/Speed" by "Sajid"
```
Comment (ActualMandM) [inline Source/Sonic Generations/Gameplay/Speed/Everything is 2x Speed Except For Sonic.hmm:1]: Except For Sonic -> Except for Sonic

File: Source/Sonic Generations/Gameplay/Speed/Everything is 4x Speed Except For Sonic.hmm [hunk 11]
```diff
@@ -1,4 +1,4 @@
-Code "Everything is 4x Speed Except For Sonic" in "Gameplay" by "Sajid" 
+Code "Everything is 4x Speed Except For Sonic" in "Gameplay/Speed" by "Sajid"
```
Comment (ActualMandM) [inline Source/Sonic Generations/Gameplay/Speed/Everything is 4x Speed Except For Sonic.hmm:1]: Except For Sonic -> Except for Sonic

File: Source/Sonic Generations/Gameplay/Speed/Everything is Half Speed Except For Sonic.hmm [hunk 12]
```diff
@@ -1,4 +1,4 @@
-Code "Everything is 1/2 Speed Except For Sonic" in "Gameplay" by "Sajid" 
+Code "Everything is 0.5x Speed Except For Sonic" in "Gameplay/Speed" by "Sajid"
```
Comment (ActualMandM) [inline Source/Sonic Generations/Gameplay/Speed/Everything is Half Speed Except For Sonic.hmm:1]: Except For Sonic -> Except for Sonic

File: Source/Sonic Generations/Gameplay/Speed/Everything is Quarter Speed Except For Sonic.hmm [hunk 13]
```diff
@@ -1,4 +1,4 @@
-Code "Everything is 1/4 Speed Except For Sonic" in "Gameplay" by "Sajid" 
+Code "Everything is 0.25x Speed Except For Sonic" in "Gameplay/Speed" by "Sajid"
```
Comment (ActualMandM) [inline Source/Sonic Generations/Gameplay/Speed/Everything is Quarter Speed Except For Sonic.hmm:1]: Except For Sonic -> Except for Sonic

File: Source/Sonic Generations/Graphics/Event/Disable Title Loading Video.hmm [hunk 14]
```diff
@@ -0,0 +1,4 @@
+Patch "Disable Title Loading Video" in "Graphics/Event" by "Hyper" does "Disables the few extra low resolution frames of Sonic falling when loading out of the title screen."
```
Comment (ActualMandM) [inline Source/Sonic Generations/Graphics/Event/Disable Title Loading Video.hmm:1]: Graphics/Event -> UI (to coincide with Disable Title Outro)

File: Source/Sonic Generations/Graphics/Post-processing/Disable Color Correction.hmm [hunk 15]
```diff
@@ -0,0 +1,4 @@
+Patch "Disable Color Correction" in "Graphics/Post-processing" by "M&M"
```
Comment (ActualMandM) [inline Source/Sonic Generations/Graphics/Post-processing/Disable Color Correction.hmm:1]: Color -> Colour

File: Source/Sonic Generations/UI/Always Display Alternate Name for Rooftop Run.hmm [hunk 16]
```diff
@@ -0,0 +1,4 @@
+Patch "Always Display Alternate Name for Rooftop Run" in "UI" by "Hyper" does "Uses the 'Orange Roofs' stage title instead of 'Rooftop Run' for White World, which is the Japanese name of the stage."
```
Comment (ActualMandM) [inline Source/Sonic Generations/UI/Always Display Alternate Name for Rooftop Run.hmm:1]: Alternate -> Japanese, UI\White Space

File: Source/Sonic Generations/UI/Always Use Original Stage Text Colours.hmm [hunk 17]
```diff
@@ -0,0 +1,4 @@
+Patch "Always Use Original Stage Text Colours" in "UI" by "Hyper" does "Uses the coloured stage title models in White World, instead of being gold if you have S ranks for those particular stages."
```
Comment (ActualMandM) [inline Source/Sonic Generations/UI/Always Use Original Stage Text Colours.hmm:1]: White World -> White Space, UI\White Space

File: Source/Sonic Lost World/Fixes/Graphics/Fix Depth Effects On Intel GPUs.hmm [hunk 18]
```diff
@@ -1,4 +1,4 @@
-Patch "Fix Depth Effects On Intel GPUs" in "Fixes" by "Skyth" 
+Patch "Fix Depth Effects On Intel GPUs" in "Fixes/Graphics" by "Skyth"
```
Comment (ActualMandM) [inline Source/Sonic Lost World/Fixes/Graphics/Fix Depth Effects On Intel GPUs.hmm:1]: Effects On Intel -> Effects on Intel

File: Source/Sonic Lost World/Fixes/Graphics/Fix Depth Of Field Draw Order.hmm [hunk 19]
```diff
@@ -1,4 +1,4 @@
-Patch "Fix Depth Of Field Draw Order" in "Fixes" by "Sajid" does "Changes the pseudo-dof implementation of the game into the proper one seen on Wii U." 
+Patch "Fix Depth Of Field Draw Order" in "Fixes/Graphics" by "Sajid" does "Changes the pseudo-dof implementation of the game into the proper one seen on Wii U."
```
Comment (ActualMandM) [inline Source/Sonic Lost World/Fixes/Graphics/Fix Depth Of Field Draw Order.hmm:1]: Depth Of Field -> Depth of Field

File: Source/Sonic Lost World/Graphics/Display/Widescreen Support.hmm [hunk 20]
```diff
@@ -1,4 +1,4 @@
-Code "Widescreen Support" in "Graphics" by "Muzzarino" does "Dynamically adjusts the aspect ratio to match the game resolution (recommended for ultrawide)." 
+Code "Widescreen Support" in "Graphics/Display" by "Muzzarino" does "Dynamically adjusts the aspect ratio to match the game resolution (recommended for ultrawide)."
```
Comment (ActualMandM) [inline Source/Sonic Lost World/Graphics/Display/Widescreen Support.hmm:1]: rename to Adjust Aspect Ratio to Resolution for consistency with other games

File: Source/Sonic Origins/Graphics/Frontend/FPS/30 FPS.hmm [hunk 21]
```diff
@@ -1,4 +1,9 @@
-Code "Locked 30 FPS for Main Menu" in "Graphics" by "MegAmi" does "Limits the main menu to running at 30 FPS. This works similarly to how the Switch version's FPS cap works, meaning the actual games are unaffected."
+Code "30 FPS" in "Graphics/Frontend/FPS" by "MegAmi" does
```
Comment (ActualMandM) [inline Source/Sonic Origins/Graphics/Frontend/FPS/30 FPS.hmm:1]: Graphics\Frontend, no need to nest this much for a category that has only one code

**Review by ActualMandM — CHANGES_REQUESTED** [review]
Putting all my changes in this message too because some are files that should be removed in the PR but weren't touched at all and GitHub's UI kinda sucks for this.

```
[Sonic Frontiers]
Graphics\Disable Framerate Limiter (experimental) // Duplicate, remove
Graphics\Disable Framerate Limiter in Cutscenes (experimental) // Duplicate, remove
Graphics\Post-processing\Disable Color Correction LUTs // Color -> Colour

[Sonic Generations]
Cheats\Hub\Always Unlocked All Stages // Hub -> White Space
Cheats\Hub\Disable Boss Key Check // Hub -> White Space
Gameplay\Objects\Disable Out Of Control Balloons // Out Of Control -> Out of Control
Gameplay\Objects\Disable Out Of Control Jump Springs // Out Of Control -> Out of Control
Gameplay\Speed\Everything is 2x Speed Except For Sonic // Except For Sonic -> Except for Sonic
Gameplay\Speed\Everything is 4x Speed Except For Sonic // Except For Sonic -> Except for Sonic
Gameplay\Speed\Everything is Half Speed Except For Sonic // Except For Sonic -> Except for Sonic
Gameplay\Speed\Everything is Quarter Speed Except For Sonic // Except For Sonic -> Except for Sonic
Graphics\Event\Disable Title Loading Video // Graphics/Event -> UI (to coincide with Disable Title Outro)
Graphics\Post-processing\Disable Color Correction // Color -> Colour
UI\Always Display Alternate Name for Rooftop Run // Alternate -> Japanese, UI\White Space
UI\Always Use Original Stage Text Colours // White World -> White Space, UI\White Space

[Sonic Lost World]
Fixes\Graphics\Fix Depth Effects On Intel GPUs // Effects On Intel -> Effects on Intel
Fixes\Graphics\Fix Depth Of Field Draw Order // Depth Of Field -> Depth of Field
Graphics\Display\Widescreen Support // rename to Adjust Aspect Ratio to Resolution for consistency with other games

[Sonic Origins]
Graphics\Frontend\FPS\30 FPS // Graphics\Frontend, no need to nest this much for a category that has only one code
```

File: Source/Sonic Frontiers/Graphics/Post-processing/Disable Color Correction LUTs.hmm (hunk 5 above)
Comment (thesupersonic16) [reply Source/Sonic Frontiers/Graphics/Post-processing/Disable Color Correction LUTs.hmm:1]:
```suggestion
Patch "Disable Colour Correction LUTs" in "Graphics/Post-processing" by "M&M" does "Prevents the game from using the color corrected LUTs stored in each stage's trr_cmn pac." 
```

File: Source/Sonic Lost World/Graphics/Event/Player Resolution/1280x720.hmm [hunk 22]
```diff
@@ -1,4 +1,4 @@
-Patch "1280x720 Movie Player" in "Graphics" by "ĐeäTh" does "Forces the CRIWARE video player to use a 1280x720 viewport. This is only useful for mods that have higher resolution video files." 
+Patch "1280x720" in "Graphics/Event/Player Resolution" by "ĐeäTh" does "Forces the CRIWARE video player to use a 1280x720 viewport. This is only useful for mods that have higher resolution video files."
```
Comment (Sajidur78) [inline Source/Sonic Lost World/Graphics/Event/Player Resolution/1280x720.hmm:1]: I don't think these have to be so deep in the category tree or need their own category. These could easily be moved into `Display/Internal Resolution` or `Display/Internal Resolution/Movie Player`.

File: Source/Sonic Origins/Graphics/Frontend/FPS/30 FPS.hmm (hunk 21 above)
Comment (thesupersonic16) [inline Source/Sonic Origins/Graphics/Frontend/FPS/30 FPS.hmm:1]:
```suggestion
Code "30 FPS" in "Graphics" by "MegAmi" does
```
Since there is only one FPS code, I don't think we should be using a FPS category, I am not really sure about Frontend either. Since there is only one code in Graphics, I think its safe to leave it in the root of Graphics.

**Review by Sajidur78 — CHANGES_REQUESTED** [review]

File: Source/Sonic Forces/Gameplay/Input/360 Degree Movement.hmm [hunk 23]
```diff
@@ -0,0 +1,4 @@
+Patch "360 Degree Movement" in "Gameplay/Input" by "Skyth"
```
Comment (thesupersonic16) [inline Source/Sonic Forces/Gameplay/Input/360 Degree Movement.hmm:1]:
```suggestion
Patch "360 Degree Movement" in "Gameplay" by "Skyth" 
```
I don't think it needs an Input folder as there is only two codes in the Gameplay folder with one per. I think leaving it in Gameplay alone makes the tree a bit less cluttered.

File: Source/Sonic Forces/Gameplay/Objects/Disable Dash Panels.hmm [hunk 24]
```diff
@@ -0,0 +1,4 @@
+Patch "Disable Dash Panels" in "Gameplay/Objects" by "SuperSonic16"
```
Comment (thesupersonic16) [inline Source/Sonic Forces/Gameplay/Objects/Disable Dash Panels.hmm:1]:
```suggestion
Patch "Disable Dash Panels" in "Gameplay" by "SuperSonic16" 
```
ditto

File: Source/Sonic Frontiers/Graphics/Post-processing/Disable Color Correction LUTs.hmm (hunk 5 above)
Comment (Sajidur78) [reply Source/Sonic Frontiers/Graphics/Post-processing/Disable Color Correction LUTs.hmm:1]: Consider renaming this patch as the name is incredibly misleading. It assumes the effect disabling colour correction entirely whereas all it does is disable level specific LUTs. A more appropriate name would be `Disable Level Specific LUTs` or `Disable Level Specific Colour Correction LUTs`. Personally, I'm leaning towards the shorter variant.

File: Source/Sonic Forces/Cheats/Player/Disable Ring Invulnerability.hmm [hunk 25]
```diff
@@ -0,0 +1,4 @@
+Patch "Disable Ring Invulnerability" in "Cheats/Player" by "SuperSonic16" does "Disables the player being invulnerable with at least one ring."
```
Comment (thesupersonic16) [inline Source/Sonic Forces/Cheats/Player/Disable Ring Invulnerability.hmm:1]: Can't really give a suggestion here, but while this title does make sense, I am not so sure if its easy for others to understand "Ring Invulnerability". Could there be a better title to use here? if not, leave it as is.

**Review by thesupersonic16 — CHANGES_REQUESTED** [review]
There are four codes that I think needs attention/changes, I have made comments to the file.

File: Source/Sonic Origins/Graphics/Frontend/FPS/30 FPS.hmm (hunk 21 above)
Comment (ActualMandM) [reply Source/Sonic Origins/Graphics/Frontend/FPS/30 FPS.hmm:1]: The name after this change will imply it makes the entire game 30FPS when in reality it's just the HE2 menu that gets set.

File: Source/Sonic Frontiers/Graphics/Post-processing/Disable Color Correction LUTs.hmm (hunk 5 above)
Comment (ActualMandM) [reply Source/Sonic Frontiers/Graphics/Post-processing/Disable Color Correction LUTs.hmm:1]: The shorter one does sound better for what this code does.

File: Source/Sonic Origins/Graphics/Frontend/FPS/30 FPS.hmm (hunk 21 above)
Comment (thesupersonic16) [reply Source/Sonic Origins/Graphics/Frontend/FPS/30 FPS.hmm:1]: Fair, either state that in the code or drop it to Frontend or Main Menu

File: Source/Sonic Generations/Cheats/Player/Disable Life Subtraction on Restart.hmm [hunk 26]
```diff
@@ -0,0 +1,3 @@
+Patch "Disable Life Subtraction on Restart" in "Cheats/Player" by "M&M" 
+WriteProtected<byte>(0xD06919, 0xEB, 0x29);
```
Comment (blueskythlikesclouds) [inline Source/Sonic Generations/Cheats/Player/Disable Life Subtraction on Restart.hmm:2]: I would call this "Disable Life Loss on Restart" personally.

File: Source/Sonic Generations/Gameplay/Player/Subtract Super Sonic Rings Every Second.hmm [hunk 27]
```diff
@@ -1,4 +1,4 @@
-Patch "Decrease Super Sonic Rings Every Second" in "Gameplay" by "brianuuu and M&M" 
+Patch "Subtract Super Sonic Rings Every Second" in "Gameplay/Player" by "brianuuu and M&M"
```
Comment (blueskythlikesclouds) [inline Source/Sonic Generations/Gameplay/Player/Subtract Super Sonic Rings Every Second.hmm:1]: I think this actually sounded better with "Decrease"

File: Source/Sonic Generations/Graphics/Quality/Disable Depth of Field.hmm [hunk 28]
```diff
@@ -0,0 +1,4 @@
+Code "Disable Depth of Field" in "Graphics/Quality" by "Luig"
```
Comment (blueskythlikesclouds) [inline Source/Sonic Generations/Graphics/Quality/Disable Depth of Field.hmm:1]: Quality -> Post-processing

File: Source/Sonic Generations/Graphics/Quality/Disable Light Shafts.hmm [hunk 29]
```diff
@@ -0,0 +1,4 @@
+Code "Disable Light Shafts" in "Graphics/Quality" by "Skyth"
```
Comment (blueskythlikesclouds) [inline Source/Sonic Generations/Graphics/Quality/Disable Light Shafts.hmm:1]: Quality -> Post-processing

File: Source/Sonic Generations/Graphics/Quality/Disable Motion Blur.hmm [hunk 30]
```diff
@@ -0,0 +1,4 @@
+Code "Disable Motion Blur" in "Graphics/Quality" by "Skyth"
```
Comment (blueskythlikesclouds) [inline Source/Sonic Generations/Graphics/Quality/Disable Motion Blur.hmm:1]: Quality -> Post-processing

File: Source/Sonic Generations/Graphics/Quality/Force Motion Blur.hmm [hunk 31]
```diff
@@ -1,4 +1,4 @@
-Code "Force Motion Blur" in "Graphics" by "Skyth & Kitzu" 
+Code "Force Motion Blur" in "Graphics/Quality" by "Skyth & Kitzu"
```
Comment (blueskythlikesclouds) [inline Source/Sonic Generations/Graphics/Quality/Force Motion Blur.hmm:1]: Quality -> Post-processing

File: Source/Sonic Generations/Graphics/Quality/Force Object Blur on All Player Models.hmm [hunk 32]
```diff
@@ -1,4 +1,4 @@
-Patch "Use Object Blur On All Player Models" in "Graphics" by "Skyth" 
+Patch "Force Object Blur on All Player Models" in "Graphics/Quality" by "Skyth"
```
Comment (blueskythlikesclouds) [inline Source/Sonic Generations/Graphics/Quality/Force Object Blur on All Player Models.hmm:1]: Quality -> Post-processing

File: Source/Sonic Generations/Graphics/Post-processing/Disable FxPipeline Radial Blur.hmm [hunk 33]
```diff
@@ -0,0 +1,4 @@
+Patch "Disable FxPipeline Radial Blur" in "Graphics/Post-processing" by "Hyper"
```
Comment (blueskythlikesclouds) [inline Source/Sonic Generations/Graphics/Post-processing/Disable FxPipeline Radial Blur.hmm:1]: Remove this code entirely, it does not do anything anymore

File: Source/Sonic Generations/Gameplay/Player/Subtract Super Sonic Rings Every Second.hmm (hunk 27 above)
Comment (DeaTh-G) [reply Source/Sonic Generations/Gameplay/Player/Subtract Super Sonic Rings Every Second.hmm:1]: Would it make sense to call this code `Traditional Super Ring Loss` with a description explaining that it makes the player lose 1 ring per every second, much like in every other sonic game when in Super Form?

Comment (ActualMandM) [reply Source/Sonic Generations/Gameplay/Player/Subtract Super Sonic Rings Every Second.hmm:1]: > Would it make sense to call this code `Traditional Super Ring Loss` with a description explaining that it makes the player lose 1 ring per every second, much like in every other sonic game when in Super Form?

Would need to specify that it's Super Sonic Ring Loss as we have the `Instantly Collect Super Rings` code

File: Source/Sonic Frontiers/Graphics/Post-processing/Disable Color Correction LUTs.hmm (hunk 5 above)
Comment (HyperBE32) [reply Source/Sonic Frontiers/Graphics/Post-processing/Disable Color Correction LUTs.hmm:1]: Final name will be `Disable Stage Specific LUTs`.

File: Source/Sonic Forces/Cheats/Player/Disable Ring Invulnerability.hmm (hunk 25 above)
Comment (HyperBE32) [reply Source/Sonic Forces/Cheats/Player/Disable Ring Invulnerability.hmm:1]: I see this as a non-issue, as the description explains it in full.

File: Source/Sonic Generations/Cheats/Player/Disable Life Subtraction on Restart.hmm (hunk 26 above)
Comment (HyperBE32) [reply Source/Sonic Generations/Cheats/Player/Disable Life Subtraction on Restart.hmm:2]: Renamed to `Disable Life Loss from Restart` for consistency with the "Ring Loss" codes.

**Review by ActualMandM — APPROVED** [review]
LGTM

**Review by thesupersonic16 — APPROVED** [review]
LGTM

**Review by Sajidur78 — APPROVED** [review]
LGTM
